pages/index.html

```diff
@@ -1,12 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <title> Portfolio</title>
+    <link href="/styles/style.css" rel="stylesheet" />
+    
+  </head>
+  <body>
+    <header>
+        <h1>Hi! I am  Niloofar Nayebi</h1>
+        <p class="line">Software Enginner student at Ada</p>
+        <div class="container">
+        <nav  >
+            <ul class="link">
+                <li>
+                    <a  href="about.html">about</a>
+                    &nbsp;
+                  </li>
+               
+                <li>
+                    <a href="portfolio.html">portfolio</a>
+                  </li>
+            </ul>
+        </nav>
+        </div>
+           
+        
+
+    </header>
+    
+        <img id="pic"
+  alt="A mountain. The tops of the mountain are covered in clouds and fog."
+  src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS5kZuhJAyLvlpMuajxApQLr78t6TBfmnBEMA&usqp=CAU" style="width:50%"
+/>
```

Comment:
Let's put the rest of this content inside the `main` element. That's where everything besides the header and footer should go.